assignment1/src/components/HangMan.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <v-card variant="outlined">
+        <v-card-title>HangMan</v-card-title>
+        <v-card-text>Lorem ipsum dolor sit amet, 
+            consectetur adipiscing elit. Sed a sem 
+            luctus arcu tincidunt condimentum. 
+            Ut hendrerit nisi ante, 
+            sed fermentum purus 
+            commodo tristique. Cras pharetra 
+            sodales sapien, in ultrices urna semper vel. 
+            Sed at tincidunt orci, quis fringilla purus. 
+            Quisque sed cursus ligula, quis faucibus est. 
+            Praesent nec elit nisi. 
+            Integer convallis blandit arcu eu luctus. 
+            Quisque a imperdiet sapien. 
+            Nam ac interdum ante. 
+            Aliquam dapibus neque in justo maximus, eget placerat nisl porttitor. 
+            Integer pretium ut ante maximus consectetur. 
+            ed porttitor finibus risus sit amet hendrerit. 
+            Nulla laoreet auctor ante eget bibendum.</v-card-text>
```

Comment:
```suggestion
        <v-card-text>
            Lorem ipsum dolor sit amet, 
            consectetur adipiscing elit. Sed a sem 
            luctus arcu tincidunt condimentum. 
            Ut hendrerit nisi ante, 
            sed fermentum purus 
            commodo tristique. Cras pharetra 
            sodales sapien, in ultrices urna semper vel. 
            Sed at tincidunt orci, quis fringilla purus. 
            Quisque sed cursus ligula, quis faucibus est. 
            Praesent nec elit nisi. 
            Integer convallis blandit arcu eu luctus. 
            Quisque a imperdiet sapien. 
            Nam ac interdum ante. 
            Aliquam dapibus neque in justo maximus, eget placerat nisl porttitor. 
            Integer pretium ut ante maximus consectetur. 
            ed porttitor finibus risus sit amet hendrerit. 
            Nulla laoreet auctor ante eget bibendum.
        </v-card-text>
```
Typically, the tags are on their own lines when there are large blocks of text